README.md

```diff
@@ -77,7 +83,7 @@ Deployment is normally done by CI.
 
 If you want to do a manual deployment:
 
-Configure your AWS credentials, see https://serverless.com/framework/docs/providers/aws/guide/credentials
+Configure your AWS credentials, see <https://serverless.com/framework/docs/providers/aws/guide/credentials>
```

Comment:
Just curious what does change?

Reply:
This is a linting fix for avoiding bare URL. My editor said so. 🤷‍♂️